Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -800,7 +789,7 @@ protected function copyHierarchyRelationToDimensionSpacePoint(
             $this->getRelationPosition(
                 $newParent,
                 $newChild,
-                null, // todo: find proper sibling
+                $newSucceedingSibling,
```

Comment:
so it would fix a few bugs if one does a replay? So def recommended? `flow cr:replayProjection contentGraph`?

Reply:
Nope, that will always be null for oder events